packages/admin-panel/src/table/DataFetchingTable.js

```diff
@@ -239,7 +239,7 @@ DataFetchingTableComponent.defaultProps = {
   numberOfPages: 0,
   nestingLevel: 0,
   deleteConfig: {},
-  TableComponent: null,
+  TableComponent: undefined,
```

Comment:
should be undefined instead of null so that it doesn't over write the material ui default component.